arm-network/2017-06-01/swagger/applicationGateway.json

```diff
@@ -390,7 +390,7 @@
         "tags": [
           "ApplicationGateways"
         ],
-        "operationId": "ApplicationGateways_ListAvailableSslOptions",
+        "operationId": "ApplicationGateways_AvailableSslOptions",
```

Comment:
This seems to return a list of Sub resource, am I correct? if so I think the right naming was chosen, thoughts @veronicagg & @balajikris